aten/src/ATen/native/TensorShape.cpp

```diff
@@ -3377,8 +3389,8 @@ Tensor _unsafe_view(const Tensor& self, IntArrayRef size) {
 
 Tensor unsqueeze(const Tensor& self, int64_t dim) {
```

Comment:
It seems you are symint-ify unsqueeze and unsqueeze_copy?

Reply:
Yeah.. although unsqueeze already is symint-ified.. idk which path it goes down